spec/jobs/sms_sender_number_change_job_spec.rb

```diff
@@ -1,13 +1,15 @@
 require 'rails_helper'
 
-describe SmsSenderNumberChangeJob, sms: true do
+describe SmsSenderNumberChangeJob do
   describe '.perform' do
-    it 'sends number change message to the phone number' do
+    it 'sends number change message to the mobile number', twilio: true do
```

Comment:
wellll I left it `mobile` since it's receiving SMS, but for consistency I'll update to phone